docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md

```diff
@@ -1,81 +1,89 @@
 ---
 title: Managing product offers
-description: This document describes the actions a Merchant can do in the Offers section in the Merchant Portal.
+description: This topic describes the actions a Merchant can do in the Offers section in the Merchant Portal.
 template: back-office-user-guide-template
 ---
 
-This document describes the actions a merchant can do in the **Offers** section of the Merchant Portal.
+This topic describes the actions a Merchant can do in the Offers section in the Merchant Portal.
 
 ## Prerequisites
 
-To start managing product offers, navigate to the **Merchant Portal&nbsp;<span aria-label="and then">></span> Offers**.
+To start managing product offers, navigate to the **Merchant Portal** > **Offers**.
 
-Review the reference information in this article before you start, or look up the necessary information as you go through the process.
+Review the reference information in this article before you start, or just look up the necessary information as you go through the process.
 
 ## Creating a product offer
 
-To create a product offer, take the following steps:
+To create a product offer:
```

Comment:
we should introduce procedures with the full sentence: https://developers.google.com/style/procedures